.gitignore

```diff
@@ -103,4 +103,6 @@ dist
 # TernJS port file
 .tern-port
 
-lib/
\ No newline at end of file
+lib/
+.vscode/
```

Comment:
Why ignore the .vscode folder?

Reply:
Some extensions may add on a `.vscode/settings.json` file like DeepScan, which is really unnecessary to deal with

Reply:
I've got the `.vscode/settings.json` file for autoformatting with eslint, though I could move that to my own settings.